modelplugin/aether-2.1.0/aether-2.1.0.tree

```diff
@@ -0,0 +1,152 @@
+module: access-profile
```

Comment:
I think you could put the YANG files and the "tree" in to `onos-helm-charts/config-models` - they will have to go there eventually anyway. You can always referenc them from there in the Makefile